src/engine/source/vdscanner/test/src/unit/versionMatcher_test.cpp

```diff
@@ -385,7 +385,7 @@ TEST_F(VersionMatcherTest, comparePEP440_AlternativeSyntax_OkEqual)
 
     // Implicit post releases
     EXPECT_NO_THROW((compareResult = VersionMatcher::compare("1.2.3-1", "1.2.3.post1", VersionObjectType::PEP440)));
-    EXPECT_EQ(compareResult, VersionComparisonResult::A_EQUAL_B);
+    EXPECT_EQ(compareResult, VersionComparisonResult::A_LESS_THAN_B);
```

Comment:
I'd like to know if this change is right or not.
If it isn't, what should we need to improve?
If it is, what should we backport?

Reply:
We talked about this scenario and the implicit conversion of the post-release. We agree to keep this change